### PR TITLE
docs: add warming up for kitex

### PR DESCRIPTION
**Pull request by felix021**
#### What type of PR is this?
docs: Documentation only changes

#### Check the PR title.
- [X] This PR title match the format: \<type\>(optional scope): \<description\>
- [X] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).
en:
zh(optional):

### CODE REVIEW

**Review by joway — DISMISSED** [review]

File: content/en/docs/kitex/Tutorials/basic-feature/warming_up.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Where `warmup.ClientOption` is the struct introduced by package `pkg/warmup` since version v0.3.0:
+
+```
+// ClientOption controls the warming up of a client.
+type ClientOption struct {
+    ErrorHandling
+    ResolverOption *ResolverOption
+    PoolOption     *PoolOption
+}
+```
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/warming_up.md:31]: 本文中所有代码块，使用go语言的，需要加 go 标注，即 ```go

File: content/en/docs/kitex/Tutorials/basic-feature/warming_up.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    "cluster": "default",
+                    // "env": "xxx"
+                },
+            },
+        },
+    },
+}))
+```
+
+#### 2. 2. Initialize the connection pool when creating a client, and establish two connections for each downstream instance
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/basic-feature/warming_up.md:79]: markdown 标题里面不用加数字序号，而且这里 2.2. 还重复了

File: content/en/docs/kitex/Tutorials/basic-feature/warming_up.md (hunk 1 above)
Comment (felix021) [reply content/en/docs/kitex/Tutorials/basic-feature/warming_up.md:31]: OK，我改一下，和另一个文档的pr一起提交吧

File: content/en/docs/kitex/Tutorials/basic-feature/warming_up.md (hunk 2 above)
Comment (felix021) [reply content/en/docs/kitex/Tutorials/basic-feature/warming_up.md:79]: OK

**Pull request closed without merging**